pyproject.toml

```diff
@@ -9,6 +9,15 @@ fallback_version = "999.999.999+unknown"
 [tool.coverage.run]
 omit = ["*/tests/*", "*pygmt/__init__.py"]
 
+[tool.flakeheaven]
+max_line_length = 88
+max_doc_length = 79
+show_source = true
+
+[tool.flakeheaven.plugins]
+pycodestyle = ["+*", "-E266", "-E501", "-W503"]
+pyflakes = ["+*", "-F401"]
```

Comment:
Seems like flakeheaven exceptions need to be handled here (see https://flakeheaven.readthedocs.io/en/latest/config.html#ignored-options).

For reference, original list from `setup.cfg` was `E203, E266, E501, W503, F401, E741`. The removed exceptions are `E203 W503`.

Reply:
- [`E203`](https://www.flake8rules.com/rules/E203.html): Whitespace before ':' 
- [`E266`](https://www.flake8rules.com/rules/E266.html): Too many leading '#' for block comment
- [`E501`](https://www.flake8rules.com/rules/E501.html): Line too long (82 > 79 characters)
- [`W503`](https://www.flake8rules.com/rules/W503.html): Line break occurred before a binary operator
- [`F401`](https://www.flake8rules.com/rules/F401.html): Module imported but unused
- [`E741`](https://www.flake8rules.com/rules/E741.html): Do not use variables named 'I', 'O', or 'l'

Reply:
I'm OK with removing `E203` and `E741`. 

1. Perhaps we can also remove `E266`. The only file that violate this rule is `examples/gallery/embellishments/colorbar.py`
2. For `F401`, I don't think it's a good idea to ignore all "imported-but-unused" modules. Instead, we should only ignore the rules in specific files (e.g., `**/__init__.py`). This can be done by using flakeheaven's `exceptions` settings (https://flakeheaven.readthedocs.io/en/latest/config.html#example).

Reply:
Cool, removed `E266` in 6822d87ffd2860ddf1690acf15c5e35c70798083 and made `F401` specific to `**/__init__.py` files only in 7dd65fa945b648c443b0dd03055253e54b709437!